pyads/pyads_ex.py

```diff
@@ -735,6 +779,159 @@ def adsGetHandle(port, address, data_name):
     return handle
 
 
+def adsGetSymbolInfo(port, address, data_name):
```

Comment:
There is already a function that returns the handle of a PLC variable. It is called adsGetHandle. It looks like adsGetSymbolInfo does exactly the same. So maybe you could just use adsGetHandle here.

Reply:
Sorry, I missed editing the first line of the comment. This function actually returns the Symbol Info, which includes the Index Group and Offset of the variable (needed for the sum commands) as well as the type.
I'll fix the comment